package/MDAnalysis/coordinates/DCD.py

```diff
@@ -164,8 +164,14 @@ def __init__(self, filename, convert_units=True, dt=None, **kwargs):
         self.ts.dt = dt
         warnings.warn("DCDReader currently makes independent timesteps"
                       " by copying self.ts while other readers update"
-                      " self.ts inplace. This behaviour will be changed in"
-                      " 3.0 to be the same as other readers",
+                      " self.ts inplace. This behavior will be changed in"
+                      " 3.0 to be the same as other readers. In practice this"
+                      " means that you will no longer be able to store independent"
+                      " references to the Timestep object associated with the"
+                      " DCDReader. Read more at"
+                      " https://github.com/MDAnalysis/mdanalysis/issues/3899"
```

Comment:
This is about *Bump up codecov action, add token secret* — probably wrong URL.

Reply:
Whoops sorry.